queries.sql

```diff
@@ -9,3 +9,50 @@ SELECT * from animals WHERE neutered = true;
 SELECT * from animals WHERE name != 'Gabumon';
 SELECT * from animals WHERE weight_kg <= 17.3 AND weight_kg >= 10.4;
 
+/* transactions */
+BEGIN;
+UPDATE animals
+SET species = 'unspecified'
+COMMIT;
+
```

Comment:
- [ ] Its great that you have make all species `unspecified` 😊 but you missed the second point of the previous reviewer which was that you will have to **Rollback** your transaction after setting the `species = 'unspecified'` 👍 
- [ ] Previous reviewer asked you to verify data using SELECT statement after making changes 👍

 
**After doing these changes your query will look like this ⬇️**

```sql
   BEGIN; -- start transaction

   UPDATE animals
   SET species = 'unspecified'; -- make change

   SELECT species from animals; -- verify that change was made
   ROLLBACK;

   SELECT species from animals; -- verify that change was undone
```